Documentation/network/servicemesh/ingress.rst

```diff
@@ -19,6 +19,19 @@ routing and for TLS termination.
     The ingress controller creates a service of LoadBalancer type, so
     your environment will need to support this.
 
+Cilium allows you to specify load balancer mode for the Ingress resource:
+
+- ``dedicated``: The Ingress controller will create a dedicated loadbalancer
+  for the ingress.
+- ``shared``: The Ingress controller will use a shared loadbalancer for all
+  Ingress resources.
```

Comment:
Would be nice to have a word on the trade-offs, so readers can understand what option is best suited to their use case?

Reply:
good point :100: